extractor/src/main/resources/config-template.html

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
```

Comment:
@Romani, i was supposed to convert to a html file, right? I mean i can see the extension changed.

Reply:
No, it should stay xml. Html is just relaxed xml.
Checkstyle is operating on xml, on strict format.

Reply:
> No, it should stay xml. Html is just relaxed xml. Checkstyle is operating on xml, on strict format.



> No, it should stay xml. Html is just relaxed xml. Checkstyle is operating on xml, on strict format.



> No, it should stay xml. Html is just relaxed xml. Checkstyle is operating on xml, on strict format.

if i understood it right, you wanted me to add html kind comments. i have done that. please let me know.

Reply:
I want xml comment, html comment is same.